build.gradle

```diff
@@ -21,7 +21,7 @@ task runLangtonAnt (dependsOn: 'check', type: JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 
-task runLangtonAntAdhemar (dependsOn: 'classes', type: JavaExec) {
```

Comment:
We are losing Adhemar's task here, I assume this was because of the git conflict resolution. Please add a new commit to keep both tasks, yours and Adhemar's

Reply:
Hi Jose thanks for the correction, I just added Adhermar's tasks and mine

Reply:
great, thank you!